aa_stripe/settings.py

```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8 -*-
+"""
```

Comment:
Usage of this file is wrong. If you look https://github.com/ArabellaTech/roman_api/pull/173#discussion-diff-136361453R4 it is described clearly. Please rework the settings to import from our project, not the other way.